packages/cloud_firestore/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.12.2
+
+* Ensure that all channel calls to the Dart side from the Java side are done
```

Comment:
You may want to mention that this is an Android-only change.

Reply:
Done